jekyll/_cci2/contexts.md

```diff
@@ -237,121 +237,110 @@ to set up your CircleCI command line interface._
 
 To create an environment variable using our CLI, perform the following steps:
 
-1. If you have not already done so, find the right context name that will contain the new environment variable. Execute this command in the CLI: \
+1. If you have not already done so, find the right context name that will contain the new environment variable. Execute this command in the CLI:
 `circleci context list <vcs-type> <org-name>`
-2. Store a new environment variable under that context. Execute this command in the CLI: \
-`circleci context store-secret <vcs-type> <org-name> <context-name> <secret name>`
+2. Store a new environment variable under that context. Execute this command in the CLI:
+`circleci context store-secret <vcs-type> <org-name> <context-name> <env-var-name>`
+
+Note that the CLI will prompt you to input the secret value, rather than accepting it 
+as an argument. This approach is designed to avoid unintentional secret exposure.
 
 ##### Using CircleCI’s API
 {:.no_toc}
 
-To create an environment variable using our API, perform the following steps:
-
-1. Call the [Add Environment Variable](https://circleci.com/docs/api/v2/#operation/addEnvironmentVariableToContext)
-   endpoint with the appropriate request body in one of our supported languages.
-   For this request, replace the `context-id` and the `env-var-name` with the ID
-   for the context and the new environment variable name. The request body
-   should include a `value` key with a string value.
-   
-2. If the request is successful, the response will include the name of the
-   environment variable that has been updated. An unsuccessful response will
-   include a clear error message explaining the error reported.
```

Comment:
IMO it's best to defer to API docs for this, both to keep these shorter and to avoid falling out of sync. I held onto the request schema for ease of use, but we might consider dropping that part too for the same reason.